tests/conftest.py

```diff
@@ -137,10 +137,6 @@ def minimal_attribute_dict():
         "BaseTransformer": {
             "columns": ["a"],
         },
-        "BaseTwoColumnTransformer": {
```

Comment:
Q: Why is this removed, are we not using this anymore?

Reply:
yes have refactored the logic form this transformer into two mixin classes NewColumnNameMixin and TwoColumnMixin so no longer have this base class.